ckanext/ontario_theme/templates/snippets/search_result_text.html

```diff
@@ -0,0 +1,45 @@
+{#
+
+Displays a test for results of a search.
+
+query        - The text that was searched for
+count        - The number of results for the search
+type         - Search result type (dataset, group, organization)
+
+Example:
+
+  {% snippet 'snippets/search_result_text.html', query=query, count=count, type='dataset' %}
+
+#}
+{% if type == 'dataset' %}
+  {% set text_query = ungettext('{number} dataset found', '{number} datasets found for "{query}"', count) %}
```

Comment:
I removed the {{query}} because it suggests a bit that the search is only happening on the query, not the query + filters. combined with the changes that appear below, I hope that it is effectively clearer what all the active search constraints are.